public/stylesheets/style.css

```diff
@@ -1,8 +1,240 @@
+    -webkit-transform: skewX(-22deg); /* Safari */
+    transform: skewX(-22deg);
+}
+
+#a-capital-two {
+    float: left;
+    margin-left: 3px;
+    margin-top: 98px;
+    width: 0;
+    height: 0;
+    border-top: 130px solid transparent;
+    border-left: 25px solid #47682C;
+    border-bottom: 57px solid transparent;
+    -ms-transform: skewX(-1deg); /* IE 9 */
+    -webkit-transform: skewX(-1deg); /* Safari */
+    transform: skewX(-1deg);
+}
+
+#a-capital-three {
+    float: left;
+    margin-left: -100px;
+    margin-top: 280px;
+    width: 0;
+    height: 0;
+    border-top: 188px solid transparent;
+    border-left: 40px solid #8C7051;
+    border-bottom: 0px solid transparent;
+    -ms-transform: skewX(-38deg); /* IE 9 */
+    -webkit-transform: skewX(-38deg); /* Safari */
+    transform: skewX(-38deg);
+}
+
+#a-capital-four {
+    float: left;
+    margin-left: -12px;
+    margin-top: 100px;
+    width: 10px;
+    height: 130px;
+    border-top: 0px solid transparent;
+    border-left: 50px solid #28502E;
+    border-bottom: 0px solid transparent;
+    -ms-transform: skewX(10deg); /* IE 9 */
+    -webkit-transform: skewX(10deg); /* Safari */
+    transform: skewX(10deg);
+}
+
+#a-capital-five {
+    float: left;
+    margin-left: -6px;
+    margin-top: 100px;
+    width: 20px;
+    height: 50px;
+    border-left: 0px solid transparent;
+    border-top: 129px solid #8C7051;
+    border-right: 50px solid transparent;
+    -ms-transform: skewX(10deg); /* IE 9 */
+    -webkit-transform: skewX(10deg); /* Safari */
+    transform: skewX(10deg);
+}
+
+#a-capital-six {
+    float: left;
+    margin-left: -29px;
+    margin-top: 99px;
+    width: 0px;
+    height: 0px;
+    border-left: 0px solid transparent;
+    border-bottom: 129px solid #47682C;
+    border-right: 50px solid transparent;
+    -ms-transform: skewX(-12deg); /* IE 9 */
+    -webkit-transform: skewX(-12deg); /* Safari */
+    transform: skewX(-12deg);
+}
+
+#a-capital-seven {
+    float: left;
+    margin-left: -108px;
+    margin-top: 225px;
+    width: 11px;
+    height: 0px;
+    border-left: 70px solid transparent;
+    border-top: 129px solid #28502E;
+    border-right: 0px solid transparent;
+    -ms-transform: skewX(-12deg); /* IE 9 */
+    -webkit-transform: skewX(-12deg); /* Safari */
+    transform: skewX(-12deg);
+}
+
+#a-capital-eight {
+    float: left;
+    margin-left: -54px;
+    margin-top: 226px;
+    width: 0px;
+    height: 0px;
+    border-left: 44px solid transparent;
+    border-bottom: 129px solid #8C7051;
+    border-right: 0px solid transparent;
+    -ms-transform: skewX(3deg); /* IE 9 */
+    -webkit-transform: skewX(3deg); /* Safari */
+    transform: skewX(3deg);
+}
+
+#a-capital-nine {
+    float: left;
+    margin-left: -42px;
+    margin-top: 350px;
+    width: 46px;
+    height: 118px;
+    -webkit-transform: skew(10deg);
+    -moz-transform: skew(10deg);
+    -o-transform: skew(10deg);
+    background: #47682C;
```

Comment:
thanks Professor @jeneve that's an awesome idea! I've created a new waffle card for that.

And if you want some more info on how to make those shapes check out [this post](https://medium.com/@theonlyrao/exploring-css-polygons-7a78212298b5#.6pdlkyakr) I just wrote on it :)